app/src/main/res/values/strings.xml

```diff
@@ -176,26 +176,26 @@
     <string name="vivo_keypress_workaround">vivo Origin OS keypress workaround</string>
     <string name="invalid_im">Invalid input method configuration file:\n%1$s</string>
     <string name="invalid_table_dict">Invalid dictionary:\n%1$s</string>
-    <string name="exception_table_im">Table input method configuration file (.conf or .conf.in) not found</string>
-    <string name="exception_table">Table dictionary (.dict or .txt) not found</string>
-    <string name="table_im">Table Input Methods</string>
-    <string name="manage_table_im">Manage Table Input Methods</string>
+    <string name="exception_table_im">Configuration file of table-based input method (.conf or .conf.in) not found</string>
+    <string name="exception_table">Mapping table dictionary (.dict or .txt) not found</string>
```

Comment:
To be consistent, the latter shoud be "Dictionary file of table-based input method"